commit 85dc74d8b11c8f7b17267778d616ce4f35b711b0
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-01T21:53:13+00:00 (7 months ago)
    
    TOOLS-2191 forcing smartdc_service:true prevents buildimage from building non-services

diff --git a/tools/buildimage/bin/buildimage b/tools/buildimage/bin/buildimage
index b948b0f..91d10af 100755
--- a/tools/buildimage/bin/buildimage
+++ b/tools/buildimage/bin/buildimage
@@ -506,9 +506,6 @@ function buildImage(opts, callback) {
     if (!opts.manifest.tags) {
         opts.manifest.tags = {};
     }
-    if (!opts.manifest.tags.smartdc_service) {
-        opts.manifest.tags.smartdc_service = true;
-    }
 
     if (!opts.manifest.requirements) {
         opts.manifest.requirements = {};
diff --git a/tools/mk/Makefile.defs b/tools/mk/Makefile.defs
index 81ae1b0..0eec18d 100644
--- a/tools/mk/Makefile.defs
+++ b/tools/mk/Makefile.defs
@@ -197,5 +197,6 @@ BUILDIMAGE_STAGEDIR = /tmp/buildimage-$(NAME)-$(STAMP)
 BUILDIMAGE_MF           = \
     {"name": "$(BUILDIMAGE_NAME)",\
     "description": "$(BUILDIMAGE_DESC)",\
-    "version": "$(BUILDIMAGE_VERSION)"\
+    "version": "$(BUILDIMAGE_VERSION)",\
+	"tags": {"smartdc_service": true}\
     }
